include/ccf/service/tables/security_policies.h

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+#pragma once
+
+#include "ccf/service/map.h"
+
+using DigestedPolicy = std::array<uint8_t, 32>;
+using RawPolicy = std::string;
+
+namespace ccf
+{
+  using SecurityPolicies = ServiceMap<DigestedPolicy, RawPolicy>;
+  namespace Tables
+  {
+    static constexpr auto SECURITY_POLICIES =
+      "public:ccf.gov.security_policies";
```

Comment:
I think this should be namespaced under `node`, i.e. `public:ccf.gov.nodes.security_policies` as this is about nodes (see https://microsoft.github.io/CCF/main/audit/builtin_maps.html#nodes-code-ids). 

Also, should this be `public:ccf.gov.nodes.host_data` (more generic) as the security policy is just how this field is interpreted in ACI deployments?

Reply:
I think this could go either way, I agree we should be more generic, but maybe the table cares about security policies, since that determines how we verify it (also includes the raw policy not just host data). And then the platform specific thing is that on ACI deployments, we get the digest of the security policy by reading the host_data?